README.md

```diff
@@ -0,0 +1,138 @@
+| [`phonecall`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.phonecall) | Model for phone calls in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap values. |
+| [`systemuser`](https://fivetran.github.io/dbt_dynamics_365_crm/#!/model/model.dynamics_365_crm.systemuser) | Model for system users in Dynamics 365 CRM, enriched with human-readable column names for fields with corresponding stringmap values. |
+
+### Materialized Models
+Each Quickstart transformation job run materializes 7 models if all components of this data model are enabled. This count includes all staging, intermediate, and final models materialized as `view`, `table`, or `incremental`.
+
+<!--section-end-->
+
+## How do I use the dbt package?
+
+### Step 1: Prerequisites
+To use this dbt package, you must have the following:
+
+- At least one Fivetran Microsoft Dynamics 365 CRM connection syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.
+
+#### Databricks dispatch configuration
+If you are using a Databricks destination with this package, you must add the following (or a variation of the following) dispatch configuration within your `dbt_project.yml`. This is required in order for the package to accurately search for macros within the `dbt-labs/spark_utils` then the `dbt-labs/dbt_utils` packages respectively.
+```yml
+dispatch:
+  - macro_namespace: dbt_utils
+    search_order: ['spark_utils', 'dbt_utils']
+```
+
+### Step 2: Install the package
+Include the following Microsoft Dynamics 365 CRM package version in your `packages.yml` file:
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
+```yml
+packages:
+  - package: fivetran/dynamics_365_crm
+    version: [">=0.1.0", "<0.2.0"] # we recommend using ranges to capture non-breaking changes automatically
+```
+
+### Step 3: Define database and schema variables
+
+By default, this package runs using your destination and the `dynamics_365_crm` schema. If this is not where your Microsoft Dynamics 365 CRM data is (for example, if your Microsoft Dynamics 365 CRM schema is named `dynamics_365_crm_fivetran`), add the following configuration to your root `dbt_project.yml` file:
+
+```yml
+vars:
+  dynamics_365_crm_database: your_database_name
+  dynamics_365_crm_schema: your_schema_name 
+```
+
+### Step 4: Enable/Disable Variables
+By default, this package brings in data from a select list of Microsoft Dynamics 365 CRM source tables. However, if you have disabled syncing these sources, you will need to add the following configuration to your `dbt_project.yml`:
```

Comment:
I like your approach here to not have a long list of source tables. However, we should probably link to where they can see all the source tables in case they want to understand what to turn off if necessary.

Reply:
Updated.